docs/extending/clusterprovider.md

```diff
@@ -0,0 +1,296 @@
+# Adding Cloud or Cluster Types to the Cluster Wizard
+
+The Kubernetes extension provides the Add Existing Cluster command to add a
```

Comment:
perhaps place 'Add Existing Cluster' in backticks, since it describes a command usage